lib/plugins/utils.js

```diff
@@ -47,14 +47,25 @@ export function validateStep({required}, conf) {
   return conf.length === 0 || validateSteps(conf);
 }
 
-export async function loadPlugin({cwd}, name, pluginsPath) {
+export async function loadPlugin({ cwd }, name, pluginsPath) {
   const basePath = pluginsPath[name]
     ? dirname(resolveFrom.silent(__dirname, pluginsPath[name]) || resolveFrom(cwd, pluginsPath[name]))
     : __dirname;
 
   // See https://github.com/mysticatea/eslint-plugin-node/issues/250
-  // eslint-disable-next-line node/no-unsupported-features/es-syntax
-  return isFunction(name) ? name : (await import(resolveFrom.silent(basePath, name) || resolveFrom(cwd, name))).default;
+  if (isFunction(name)) {
+    return name;
+  }
+
+  const { default: cjsExport, ...esmNamedExports } = await import(
+    resolveFrom.silent(basePath, name) || resolveFrom(cwd, name)
+  );
+
+  if (cjsExport) {
+    return cjsExport;
+  }
+
+  return esmNamedExports;
```

Comment:
nice work!

Reply:
i'm wondering, should we release this as an enhancement against v20 rather than wait for this beta to be promoted to stable? seems like it would be beneficial. maybe we try it here first by releasing as part of this upcoming beta, but could be worth backporting ahead of this becoming stable

Reply:
yeah let's do that later, if needed